modules/win-autoscale/watchmaker-win-autoscale.cfn.json

```diff
@@ -660,6 +679,43 @@
                                     ]
                                 },
                                 "waitAfterCompletion": "0"
+                            },
+                            "11-cloudwatch-agent-install": {
+                                "command": {
+                                    "Fn::If": [
+                                        "InstallCloudWatchAgent",
+                                        {
+                                            "Fn::Join": [
+                                                "",
+                                                [
+                                                    {
+                                                        "Fn::FindInMap": [
+                                                            "ShellCommandMap",
+                                                            "powershell",
+                                                            "command"
+                                                        ]
+                                                    },
+                                                    " \"& { ",
+                                                    "$CloudWatchAgentZipFile = 'c:\\cfn\\scripts\\AmazonCloudWatchAgent.zip';",
+                                                    "$CloudWatchAgentDir = 'c:\\cfn\\scripts\\AmazonCloudWatchAgent';",
+                                                    "$CloudWatchAgentInstallScript =  \"$CloudWatchAgentDir\"+'\\install.ps1';",
+                                                    "$CloudWatchAgentConfig =  \"$CloudWatchAgentDir\"+'\\aws-cloudwatch-agent-config.json';",
+                                                    "if (Test-Path $CloudWatchAgentZipFile) {",
```

Comment:
I'm somewhat concerned about false positives... I'd rather have the stack fail if this zip is not present when we are actually pulling down the zip, and so if it's not present something went sideways.